inlong-sort/sort-connectors/kafka/src/main/java/org/apache/inlong/sort/kafka/table/KafkaDynamicTableFactory.java

```diff
@@ -325,7 +374,9 @@ public DynamicTableSink createDynamicTableSink(Context context) {
                 getSinkSemantic(tableOptions),
                 parallelism,
                 inlongMetric,
-                auditHostAndPorts);
+                auditHostAndPorts,
+                innerValueDecodingFormat,
+                tableOptions.getOptional(TOPIC_PATTERN).orElse(null));
```

Comment:
Reused source option TOPIC_PATTERN for sink？How to extend it to other multiple sink connector?

Reply:
Yes i reused the 'TOPIC_PATTERN' for kafka sink, maybe you can define some common names for other multiple sink connector, such as 'database-pattern', 'table-pattern', and so on.